addons/microsoft_calendar/utils/microsoft_calendar.py

```diff
@@ -149,9 +149,11 @@ def get_events(self, sync_token=None, token=None, timeout=TIMEOUT):
         return events, next_sync_token
 
     @requires_auth_token
-    def insert(self, values, token=None, timeout=TIMEOUT):
+    def insert(self, values, token=None, timeout=TIMEOUT, need_video_call=True):
         url = "/v1.0/me/calendar/events"
         headers = {'Content-type': 'application/json', 'Authorization': 'Bearer %s' % token}
+        if values.get('isOnlineMeeting', False):
+            values['isOnlineMeeting'] = need_video_call
```

Comment:
Hi @geflx ,
This code has been added to prevent the creation of a meeting URL for events generated from the HR Holidays model.
You can find the reference PR here:https://github.com/odoo/odoo/pull/162205
Thanks :slightly_smiling_face: